docs/guides/component-testing/angular/examples.mdx

```diff
@@ -193,6 +193,192 @@ in the future
 
 :::
 
+### Testing Signals
+
+With the releases of Angular versions [17.1](https://github.com/angular/angular/blob/main/CHANGELOG.md#1710-2024-01-17) and [17.2](https://github.com/angular/angular/blob/main/CHANGELOG.md#1720-2024-02-14), [input](https://github.com/angular/angular/pull/53521) and [model](https://github.com/angular/angular/pull/54252) signals were introduced into the `@angular/core` API.
+Since signals introduced new methods and types to the core API, Cypress needed to introduce a new test harness, `@cypress/angular-signals`, in order to support the API without breaking changes.
+
+Though basic signals were introduced in Angular `16`, this testing harness requires Angular `17.2` and above.
+
+For our below examples, we will be working with a very simple component called `TestComponent`, which looks something like this:
+
+```typescript
+// app/components/test-component.component.ts
+import { Component, input, model } from '@angular/core'
+
+@Component({
+  selector: 'test-component',
+  templateUrl: './test-component.component.html',
+  standalone: true,
+})
+export class TestComponent {
+  title = input.required<string>()
+  count = model.required<number>()
+}
+```
+
+```html
+<!-- app/components/test-component.component.html -->
+<p data-cy="test-component-title-display">{{ title() }}</p>
+<p data-cy="test-component-count-display">{{ count() }}</p>
+<button data-cy="test-component-count-incr" (click)="count.set(count() + 1)">
+  Increase
+</button>
+<button data-cy="test-component-count-decr" (click)="count.set(count() - 1)">
+  Decrease
+</button>
+```
+
+#### Testing Input Signals
+
+Having `input()` signals work outside the Angular injection context is a bit tricky. Input signals cannot be created outside the Angular injection context, nor is there a way to provide an injection context. Because of this, there are two ways to test `input()` signals with Cypress Component Testing.
+
+##### Infered Generic Type
+
+In the example below, the `title` prop being passed into our `TestComponent` is a generic type of the `input()` signal. In this example, that generic type is a `string`. Cypress wraps the generic value in a writable `signal()` and merges it into the prop. In other words:
+
+```typescript
+let titleProp = 'Test Component'
+cy.mount(TestComponent, {
+  componentProperties: {
+    title: titleProp,
+  },
+})
+```
+
+Under the hood in the `@cypress/angular-signals` test harness is really:
+
+```typescript
+cy.mount(TestComponent, {
+  componentProperties: {
+    title: signal('Test Component'),
+  },
+})
+```
+
+##### Writable Signal
+
+Since the infered generic type in our example above lives outside the Angular context, there is no way for Cypress to enforce the one-way databinding on context updates. In other words, updates to the `titleProp` variable will not update the prop being passed into `TestComponent`.
```

Comment:
```suggestion
Since the inferred generic type in our example above lives outside of the Angular context, there's no way for Cypress to enforce the one-way data binding on context updates. In other words, updates to the `titleProp` variable will not update the prop being passed into `TestComponent`.
```

Reply:
addressed in 1055c7068